spec/requests/api/actions_spec.rb

```diff
@@ -58,127 +58,22 @@
-        expect(sqs_client).to have_received(:send_message).with(country_field_set_as('Macedonia'))
-      end
-
-      it 'successfully posts Micronesia to the queue' do
-        message_body[:params][:country] = 'Micronesia'
-        params[:country] = 'FM'
-        post "/api/pages/#{page.id}/actions", params
-        expect(sqs_client).to have_received(:send_message).with(country_field_set_as('Micronesia'))
-      end
-
-      it 'successfully posts Moldova to the queue' do
-        message_body[:params][:country] = 'Moldova'
-        params[:country] = 'MD'
-        post "/api/pages/#{page.id}/actions", params
-        expect(sqs_client).to have_received(:send_message).with(country_field_set_as('Moldova'))
-      end
-
-      it 'successfully posts Palestine to the queue' do
-        message_body[:params][:country] = 'Palestine'
-        params[:country] = 'PS'
-        post "/api/pages/#{page.id}/actions", params
-        expect(sqs_client).to have_received(:send_message).with(country_field_set_as('Palestine'))
-      end
-
-      it 'successfully posts Russia to the queue' do
-        message_body[:params][:country] = 'Russia'
-        params[:country] = 'RU'
-        post "/api/pages/#{page.id}/actions", params
-        expect(sqs_client).to have_received(:send_message).with(country_field_set_as('Russia'))
-      end
-
-      it 'successfully posts Saint Martin to the queue' do
-        message_body[:params][:country] = 'Saint Martin'
-        params[:country] = 'MF'
-        post "/api/pages/#{page.id}/actions", params
-        expect(sqs_client).to have_received(:send_message).with(country_field_set_as('Saint Martin'))
-      end
-
-      it 'successfully posts Sint Maarten to the queue' do
-        message_body[:params][:country] = 'Sint Maarten'
-        params[:country] = 'SX'
-        post "/api/pages/#{page.id}/actions", params
-        expect(sqs_client).to have_received(:send_message).with(country_field_set_as('Sint Maarten'))
-      end
-
-      it 'successfully posts Syria to the queue' do
-        message_body[:params][:country] = 'Syria'
-        params[:country] = 'SY'
-        post "/api/pages/#{page.id}/actions", params
-        expect(sqs_client).to have_received(:send_message).with(country_field_set_as('Syria'))
-      end
-
-      it 'successfully posts Tanzania to the queue' do
-        message_body[:params][:country] = 'Tanzania'
-        params[:country] = 'TZ'
-        post "/api/pages/#{page.id}/actions", params
-        expect(sqs_client).to have_received(:send_message).with(country_field_set_as('Tanzania'))
-      end
-
-      it 'successfully posts Venezuela to the queue' do
-        message_body[:params][:country] = 'Venezuela'
-        params[:country] = 'VE'
-        post "/api/pages/#{page.id}/actions", params
-        expect(sqs_client).to have_received(:send_message).with(country_field_set_as('Venezuela'))
+      CountriesExtension::COUNTRIES.each do |code, name|
```

Comment:
That seems like a reasonable distinction to me. I would've been fine doing this the first time but I'd thought we didn't want to do that. :)